examples/experiments/progress_bar.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+================
+ProgressBar demo
+================
+
+This example shows how to display progress between trials using
+:class:`expyfun.visual.ProgressBar`.
+"""
+from expyfun import ExperimentController, building_doc
+from expyfun.visual import ProgressBar
+import numpy as np
+
+n_trials = 6
+
+with ExperimentController('name', version='dev', window_size=[500, 500],
+                          full_screen=False, session='foo',
+                          participant='foo') as ec:
+
+    # initialize the progress bar
+    pb = ProgressBar(ec, [0, -.1, 1.5, .1], units='norm')
+
+    ec.screen_prompt('Press the number shown on the screen. Start by pressing'
+                     ' 1.', font_size=12, live_keys=[1])
```

Comment:
[Still hanging](https://circleci.com/gh/maddycapp27/expyfun/157?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

Usually instead of a bunch of conditionals, we do a single one:
```
max_wait = 0.1 if building_doc else np.inf
```
And then pass `max_wait` to all `screen_prompt` (like here) or `wait_one_press` calls.